internal/wireguard/wg.go

```diff
@@ -55,17 +59,20 @@ func Create(apiClient *api.Client, org *api.Organization, regionCode, name strin
 		if err != nil {
 			return nil, err
 		}
-		host, _ := os.Hostname()
+		emailSlug := cleanDNSPattern.ReplaceAllString(user.Email, "-")
+
+		host, err := os.Hostname()
+		if err != nil {
+			return nil, err
+		}
+		hostSlug := cleanDNSPattern.ReplaceAllString(strings.Split(host, ".")[0], "-")
```

Comment:
I think `host` can be an empty string - would you mind taking the first part of `host` only if `host` isn't empty?

Reply:
No need, the next version of the CLI ensures a hostname is present. For now, and just for _not_ having the possibility of a panic in `strings.Split(host, ".")[0]` just assign to hostSlug only if `host` isn't empty.

Reply:
`strings.Split` would not panic on an empty string, [example](https://go.dev/play/p/J-gNDibfcZM)

Reply:
Sorry, meant the `[0]` part but obviously you still get back a non-empty slice (containing at least 1 element). Sorry for the brain outage!